app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> bool:
+    try:
+        command, file, new_file = command.split(" ")
```

Comment:
It's not the best practice to change the value of function argument

Reply:
Oh yes, thx